lib/dgram.js

```diff
@@ -222,7 +222,7 @@ Socket.prototype.bind = function(port_, address_ /*, callback*/) {
 
   // resolve address first
   this._handle.lookup(address, (err, ip) => {
-    if (err) {
+    if (err) {          
```

Comment:
nit: extra whitespace added at the end here.

Reply:
You can have git remove this for you automatically. It looks like there is whitespace in other places too. If you take care of the problem now, we won't have commits removing unrelated whitespace in the future.  😄

Reply:
Weird how that got in there sorry, fixed in 305693a.